README.md

```diff
@@ -140,6 +140,106 @@ StatusCode = {
 };
 ```
 
+## Create raw gRPC requests
+
+Users can use SDK to create raw gRPC requests in byte arries (JavaScript Uint8Array) too.
+The functions have different names but the parameters are exactly identical.
+
+### Register the certificate
+```javascript
+const binary = await ClientService.createSerializedCertificateRegistrationRequest();
+```
+
+### Register contracts
+```javascript
+const binary = await clientService.createSerializedContractRegistrationRequest('contractId', 'com.example.contract.contractName', contractUint8Array, propertiesObject);
+```
+
+### Register functions
+```javascript
+const binary = await clientService.createSerializedFunctionRegistrationRequest('functionId', 'com.example.function.functionName', functionUint8Array);
+```
+
+### List registered contracts
+```javascript
+const binary = await clientService.createSerializedContractsListingRequest();
+```
+
+### Execute a contract
+```javascript
+const binary = await clientService.createSerializedContractExecutionRequest('contractId', argumentObject);
+```
+
+### Validate an asset
+```javascript
+const binary = await clientService.createSerializedLedgerValidationRequest('assetId');
+```
+
+## Send the raw gRPC requests to Scalar DL servers
+The SDK provides another `ClientServiceWithBinary` class whereby users can send byte arries of requests to Scalar DL servers.
+It is similar with `ClientService` class to construct `ClientServiceWithBinary` class.
+However, ClientServiceWithBinary is different from ClientService.
+ClientService encapsulates responses into Scalar defined objects but ClientServiceWithBinary returns gRPC native objects.
```

Comment:
Oh ,I think they should return the same object even if serialized byte array is an input.